src/_utils.ts

```diff
@@ -46,6 +47,13 @@ export function debug (...args) {
   }
 }
 
+export async function sendFetch (url: string, options?: RequestInit) {
+  const proxy = process.env.HTTPS_PROXY || process.env.https_proxy || process.env.HTTP_PROXY || process.env.http_proxy
+  const agent = createHttpsProxyAgent(proxy)
```

Comment:
Doesn't this trow when `proxy` is falsy?

Reply:
Fixed.